tests/dummy/app/acceptance/controller.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+  componentPath: Ember.computed('model.component', function() {
```

Comment:
For sure. I've seen a couple different suggestions for future-proofing as some of the details around Ember 2.0 have solidified, but haven't totally kept up. Is the "fake `attrs` hash" approach still the preferred one?

```js
// controller.js
export default Ember.Controller.extend({ attrs: {} });

// route.js
export default Ember.Route.extend({
  setupController(controller, model) {
    controller.set('attrs.componentPath', /* ... */);
  }
});
```
```hbs
{{! template.hbs }}
{{component attrs.componentPath}}
```